src/components/ProgressBar/ProgressBar.js

```diff
@@ -11,21 +11,30 @@ type Props = {
   stiffness: number,
   damping: number,
   colors: Array<string>,
+  position: 'top' | 'inline',
 };
 
 class ProgressBar extends Component<Props> {
   static defaultProps = {
+    position: 'inline',
```

Comment:
I don't think `position: inline` is a thing. Did you mean `display: inline`?

Reply:
Maybe the naming here is a bit off. It's not a css style. It's intended as a switch prop between having the progress bar inline or on the top edge of the screen.
I'd like to have it on the top edge for the loading screen.